### PR TITLE
Update dependecies.

### PR DESCRIPTION
**Pull request by XhmikosR**
/CC @cvrebert

### CODE REVIEW

File: js/.jscsrc [hunk 1]
```diff
@@ -28,7 +28,7 @@
   "requireSpaceAfterLineComment": true,
   "requireSpaceBeforeBinaryOperators": ["+", "-", "/", "*", "=", "==", "===", "!=", "!==", ">", "<", ">=", "<="],
   "requireSpaceBetweenArguments": true,
-  "requireSpacesInAnonymousFunctionExpression": { "beforeOpeningCurlyBrace": true, "beforeOpeningRoundBrace": true },
```
Comment (cvrebert) [inline js/.jscsrc]: Was the rule removed from JSCS itself, or do we not currently comply with it?

Comment (XhmikosR) [inline js/.jscsrc]: It just fails but I'm not sure if it's a regression or intentional.

**Comment by XhmikosR** [comment]
*bump*

**Comment by cvrebert** [comment]
Perhaps we could punt JSCS to another PR/issue for further investigation, and merge everything else?

**Comment by XhmikosR** [comment]
I can split it if you think it's better sure.

**Comment by XhmikosR** [comment]
Done.

**Comment by cvrebert** [comment]
:shipit: as soon as Travis passes.

**Pull request merged**